dotnet-471-dev-pack/test/inspec/controls/command_exists.rb

```diff
@@ -0,0 +1,37 @@
+path_prefix = os.family == "windows" ? 'c:' : ''
```

Comment:
Now that Pester is working, I would delete the InSpec bits from this PR.
You could possibly cut a new branch so you have a snapshot of this branch that we can refer back to, to demonstrate in future a Pester vs InSpec comparison.

Reply:
inspec code removed and snapshotted